frontend/src/constants.js

```diff
@@ -17,6 +17,8 @@ export const POST_CREATE_URL = `${BASE_URL}/post/create`;
 export const POST_GET_ALL_URL = `${BASE_URL}/post/all`;
 export const POST_COMMENT_URL = `${BASE_URL}/post/comment`;
 export const POST_UNCOMMENT_URL = `${BASE_URL}/post/uncomment`;
+export const USER_SUGGESTIONS = `${BASE_URL}/suggestions`;
+export const GET_USER_DETAILS = `${BASE_URL}/user/details?user_id=`;
```

Comment:
Can we change the variable name to have a `_URL` at the end? (for consistency)
`USER_SUGGESTIONS_LIGHT_URL` or `GET_USER_DETAILS_URL`?

Reply:
Done